Service/DRCService.php

```diff
@@ -267,7 +267,7 @@ public function drcReleaseHandler(array $data, array $configuration): array
             $this->entityManager->flush();
 
             $this->data['response'] = new Response(
-                \Safe\json_encode($objectEntity->toArray()),
+                \Safe\json_encode($objectEntity->toArray(['embedded' => true])),
```

Comment:
You have tested this and it works? 
I am not sure, but it might also be the case that we set some configuration for showing/hiding embedded elsewhere 🤔

Reply:
This works, it is also in line with how the arrays are build in the requestService ;)

Reply:
It might be confusing how I described it:
The problem was that the objects that should have been an url now were embedded in the object itself instead of showing the url, because embedded should be true in the toArray configuration for that setting to take effect.

Reply:
Ahaa, i see it now, the switch is only reached if this embedded is set to true